g9frontend/src/components/WordSearch.tsx

```diff
@@ -0,0 +1,201 @@
+            flexDirection: 'column',
+            height: '100%',
+            backgroundColor: 'white',
+            padding: '16px 32px',
+        };
+
+        const headerStyle: React.CSSProperties = {
+            padding: '20px',
+            backgroundColor: 'blue',
+            display: 'flex',
+            justifyContent: 'space-between',
+            marginBottom: '8px',
+            color:"white",
+
+
+
+        };
+
+        const rightContainerStyle: React.CSSProperties = {
+            display: 'flex',
+            gap: '8px',
+        };
+
+        const contentStyle: React.CSSProperties = {
+            display: 'flex',
+            flexGrow: 1,
+            position: 'relative',
+            height: '100%',
+        };
+
+        const leftContainerStyle: React.CSSProperties = {
+            backgroundColor: 'orange',
+            marginRight: '8px',
+            width: '30%',
+            whiteSpace: 'pre-line'
+        };
+
+        const rightContentStyle: React.CSSProperties = {
+            backgroundColor: 'pink',
+            display: 'flex',
+            flexGrow: 1,
+            alignItems: 'flex-end',
+            justifyContent: 'space-around',
+        };
+
+        const semicircleStyle: React.CSSProperties = {
+            width: '400px',
+            height: '130px',
+            backgroundColor: 'orange',
+            position: 'absolute',
+            top: '0',
+            borderRadius: '0 0 250px 250px',
+
+            display: 'flex',
+            justifyContent: 'center',  // Center horizontally
+            alignItems: 'center',      // Center vertically
+            textAlign: 'center',       // Center text horizontally within the container
+            lineHeight: '130px',
+
+
+        };
+
+        const squareInstagram: React.CSSProperties = {
+            width: '300px',
+            height: '600px',
+            marginTop: '150px',
+            backgroundColor: 'violet',
+            marginBottom: '8px',
+            color:'white',
+            whiteSpace: 'pre-line'
+        };
+
+        const squareFacebook: React.CSSProperties = {
+            width: '300px',
+            height: '600px',
+            marginTop: '150px',
+            backgroundColor: 'blue',
+            marginBottom: '8px',
+            color:'white',
+        };
+
+        const squareX: React.CSSProperties = {
+            width: '300px',
+            height: '600px',
+            marginTop: '150px',
+            backgroundColor: 'black',
+            marginBottom: '8px',
+            color:'white',
+        };
+
+        const footerStyle: React.CSSProperties = {
+            backgroundColor: 'green',
+            marginTop: '8px',
+            padding: '8px',
+            display: 'flex',
+            justifyContent: 'space-between',
+            color:'white',
+        };
+        const [text, setText] = React.useState('');
+    return (
+        <div style={containerStyle} className="root-container">
```

Comment:
You could add a folder called componentbase where you add a template for a plain component so when the tasks are assigned everyone has a starting component in which they  can solve their task. Each component should have a small size in terms of lines of code (you could specify this as a project rule). Making small components such as RootContainer that contain other components and so on will allow us to scale the project and ensure high cohesion and low coupling.